toplevel/opttoploop.ml

```diff
@@ -217,7 +218,8 @@ let execute_phrase print_outcome ppf phr =
       let oldenv = !toplevel_env in
       incr phrase_seqid;
       phrase_name := Printf.sprintf "TOP%i" !phrase_seqid;
-      Compilenv.reset ?packname:None !phrase_name;
+      Compilenv.reset ~source_provenance:Timings.Startup
```

Comment:
I actually left a comment for @mshinwell or @chambart about that on my other PR.
I hope to get some feedback from them on this.